From c8ef5cfd02de7589365eba715be47188dc0b992e Mon Sep 17 00:00:00 2001
From: Chaitanya Godavarthi <chaitanya.kiran@netexperience.com>
Date: Mon, 21 Dec 2020 06:16:32 +0530
Subject: [PATCH] firewall: Add firewall rule for interapcomm

Add firewall rule for UCC (video/voice detect messages)
and APC (Access Point Coordinator)
---
 .../config/firewall/files/firewall.config        | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/package/network/config/firewall/files/firewall.config b/package/network/config/firewall/files/firewall.config
index 58ca162cc7..f23e4594d8 100644
--- a/package/network/config/firewall/files/firewall.config
+++ b/package/network/config/firewall/files/firewall.config
@@ -129,6 +129,22 @@ config rule
 	option proto		udp
 	option target		ACCEPT
 
+config rule
+	option name		Allow-APC
+	option src		wan
+	option proto		udp
+	option dst_port		50010
+	option target		ACCEPT
+	option family		ipv4
+
+config rule
+	option name		Allow-UCC
+	option src		wan
+	option proto		udp
+	option dst_port		50000
+	option target		ACCEPT
+	option family		ipv4
+
 # include a file with users custom iptables rules
 config include
 	option path /etc/firewall.user
-- 
2.25.1

